test/table.js

```diff
@@ -0,0 +1,113 @@
+import { expect, test } from 'vitest'
+import { spaceBase } from '#theme';
+import { setup } from './_helpers.js'
+
+setup()
+
+test('border-collapse', async (t) => {
+  const classes = ["border-collapse", "border-separate"]
+
+  const { css } = await t.uno.generate(classes)
+
+  expect(css).toMatchInlineSnapshot(`
+    "/* layer: default */
+    .border-collapse{border-collapse:collapse;}
+    .border-separate{border-separate:separate;}"
+  `)
+})
+
+test('border-collapse', async (t) => {
+  const classes = ["table-auto", "table-fixed"]
+
+  const { css } = await t.uno.generate(classes)
+
+  expect(css).toMatchInlineSnapshot(`
+    "/* layer: default */
+    .table-auto{table-layout:auto;}
+    .table-fixed{table-layout:fixed;}"
+  `)
+})
+
+test('border-spacing', async (t) => {
+  const classes = spaceBase.map((spacingUnit) => {
+    return [`border-spacing-x-${spacingUnit}`, `border-spacing-y-${spacingUnit}`, `border-spacing-${spacingUnit}`];
+  }).flat()
+
+  const { css } = await t.uno.generate(classes)
+
+  expect(css).toMatchInlineSnapshot(`
```

Comment:
Maybe a normal snapshot for something this long?

We could have a quick discussion if wanted on how long a 'snapshot' to prefer inline vs. not!

Reply:
Yeah, we just went with inline during the mob, but a regular one most def makes more sense :)